### PR TITLE
Updating Footer

### PR DESCRIPTION
**Pull request by williycole**
Adding in the necessary changes to include a link to code connectors youtube account in the footer.

### CODE REVIEW

**Comment by rjbeers** [comment]
This pull request changes a lot more that it needs to in order to add the Youtube link. It looks like your editor formatted the code for the page. I admit that the formatting on this file is poor, but let's resolve that separately.

Please update this pull request to only change the footer section.

**Comment by williycole** [comment]
Should be good to go know, let me know if I missed anything.

File: _layouts/default.html [hunk 1]
```diff
@@ -170,10 +170,15 @@ <h1 class="text-light">Send word our way</h1>
                   <i class="fas fa-calendar-alt"></i>
                 </a>
               </li>
+              <li class="list-inline-item">
+                <a href="https://www.youtube.com/channel/UCILxIdh3uyw64JjtdRmdtVg" target="_blank" rel="noopener" aria-label="Code Connector's Youtube">
```
Comment (rjbeers) [inline _layouts/default.html:174]: nitpick: YouTube is actually the correct capitalization

Comment (williycole) [reply _layouts/default.html:174]: Got me, thanks this was a great learning experiance for me!

**Pull request closed without merging**